### DISCUSSION TITLE
Randomness of DNA methylation, OCU affects polyamine metabolism via SAM

### DISCUSSION THREAD

**Discussion by KestUrba** [wikipathways/pathway-curation-tasks#31] [General]
### Discussion  (WP241)

1. According to  Song JZ, Stirzaker C, Harrison J, Melki JR, Clark SJ. Hypermethylation trigger of the glutathione-S-transferase gene (GSTP1) in prostate cancer cells. Oncogene. 2002 Feb 7;21(7):1048-61. doi: 10.1038/sj.onc.1205153. PMID: 11850822 and according to my own theoretical preprint (1991) DNA methylation pattern is stochastic. Is it possible to stress this in the drawing?

2. SAM is a donor for polyamine metabolism too, which could be intensified in cancer cell - one more green remark is neccesary!

### Should this pathway be deleted?

- [ ] Pathway should be deleted

**Comment by KestUrba** [comment]
There is the hint in WikiPath about SAM and polyamines [https://www.wikipathways.org/pathways/WP290.html](url)

**Reply by danidi** [reply]
https://www.wikipathways.org/pathways/WP290.html is from Saccharomyces cerevisiae, was this pathway reported for humans as well?

**Comment by danidi** [comment]
1) This could be done with a label in the pathway and/or a comment in one of the data nodes. 
2) Would this be S-adenosylmethionine ----> polyamine metabolism? Green remarks are ideally pathways, is there a corresponding pathway available? Do you have a reference that could be added to this step of the pathway?

Do you want to add those changes to WikiPathways yourself? In this case, you would also be included in the list of co-authors of the pathway.

**Reply by KestUrba** [reply]
Thanks. PubMed search  gives 253 references starting with Lu SC.
Int J Biochem Cell Biol. 2000 Apr;32(4):391-5. doi: 10.1016/s1357-2725(99)00139-9.
PMID: 10762064 Review.
" **SAM** is utilized by three key metabolic pathways: transmethylation; transsulfuration; and polyamine synthesis. ...Polyamines are required for normal cell growth. Given the importance of SAM in tissue function, it is not surprising that this molecule is …"

I'm newcomer  - started today having experience in a biological-molecular  research of cancer  from 1987. I haven't any experience about pathways editing so, please, fulfill drawings.

**Comment by danidi** [comment]
I've added the connection to polyamine synthesis, it is already visible here: https://classic.wikipathways.org/index.php/Pathway:WP241

For the stochastic process, can you please suggest a wording and where to place it as comment? I don't have access to the article to read it in more detail.

**Reply by KestUrba** [reply]
thanks, Danidi. We go forward!

**Comment by khanspers** [comment]
@danidi : thanks for making the updates to the pathway!
@KestUrba : At this point, there is no formalized way of representing that a process is stochastic in the diagram, and the best option would be a simple graphical label as well as adding the information to the pathway description. However, Im not sure that it belongs in this  pathway, since this describes a basic biological process, and the hypermethylation you mention is relevant for specific diseases, for example cancer. Maybe the information belongs in a cancer-focused pathway that already mentions DNA methylation.

**Reply by KestUrba** [reply]
Thanks.  DNA methylation is a fundamental biological process controlling transcription of genes. When it was discovered, there started a new branch of biology - Epigenetics.  Methylation patterns differ when comparing two humans. This is the first appearance of natural randomness.  In embryo cell DNA is globally hypomethylated, in a cancer cell some regions appear to be hypermethylated when globally there is hypomethylation.  "DNA methylation is at the root of carcinogenesis".  Rats fed by methyl deficient diets without any carcinogene developed liver tumours (Poirier et al...)!
"DNA methylation marks are introduced by de novo DNA methyltransferases DNMT3a/b and are maintained throughout cell divisions by DNA methyltransferase 1 (DNMT1), which adds methyl groups to hemimethylated CpG dinucleotides generated during DNA replication". So, in a human cell when it divides there exists maintenance - precise copy of methylation pattern appears but this is not always true.
"We conduct a systematic assessment of the age-dependent methylation by the signal-to-noise ratio and identify a wealth of "deterministic" CpG probes (about 90%), whose methylation variability likely originates due to genetic and general environmental factors. The remaining 10% of "stochastic" CpG probes are arguably governed by the biological noise or incidental environmental factors. " - by
**Vershinina O, Bacalini MG, Zaikin A, Franceschi C, Ivanchenko M. Disentangling age-dependent DNA methylation: deterministic, stochastic, and nonlinear. Sci Rep. 2021 Apr 28;11(1):9201. doi: 10.1038/s41598-021-88504-0. PMID: 33911141; PMCID: PMC8080842.**
For example, when human age is  more than 65-70 there often appear  more and more differences in methylation among previous and a new cell. Sometimes this is fatal because promoters of some oncogenes appear to be not methylated - oncogenes are not "locked" enough. This is the reason why cancer starts sometimes naturally without any mutagenic affection like pollution or some radiation... So, cancer in this case  appears to be a natural disease because of human age too, but this appears in a random way - second face of randomness. Of course, this is the most important for cancer evolution understanding and could be described in a PancCanNet Community , for example. (I see the lack of all kinds of  Cancer pathways community. After more than thirty years of research  I'm constructing the Genomic architectonic theory (GATT) and have  General model of carcinogenesis but consisting lots of scenario and schedules! ) The best way to stress randomness of DNA methylation possibly could be an explanation of this phenomena in the text below the drawing plus corresponding sign.  It was discussed in **Feinberg AP. Epigenetic stochasticity, nuclear structure and cancer: the implications for medicine. J Intern Med. 2014 Jul;276(1):5-11. doi: 10.1111/joim.12224. PMID: 24635672; PMCID: PMC4065203** too. Please, cite these articles in a WikiPathways because this is a recent fundamental knowledge

**Reply by KestUrba** [reply]
one more ref.
Field AE, Robertson NA, Wang T, Havas A, Ideker T, Adams PD. DNA Methylation Clocks in Aging: Categories, Causes, and Consequences. Mol Cell. 2018 Sep 20;71(6):882-895. doi: 10.1016/j.molcel.2018.08.008. PMID: 30241605; PMCID: PMC6520108.

**Reply by danidi** [reply]
Thank you for the information. However, I agree with @khanspers that this might be too much information for this existing pathway, where the focus is somewhere else. It might be best to create a new pathway which focuses on this, e.g. something like "DNA methylation in aging" or "DNA methylation in cancer", rather than "hiding" this in the pathway of one-carbon metabolism.

**Reply by KestUrba** [reply]
Thanks for your opinion. One Carbon unit metabolism has the main OUTPUTS: 1. DNA methylation via Methionine cycle (- the origin of Epigenetics, target for cancer therapies )  2 . DNA nucleotides synthesis via folate system (target for cancer therapies too) 3. Polyamines synthesis (target for cancer therapies)...
The more detailed connections WikiPathways between molecular and biological levels will present, the more valuable information will help users to fight diseases.  Stochasticity of DNA methylation is fundamental feature of diseases epigenetical  initiation. We must shout about it!  Especially because the headlines like this One-carbon Metabolism at the Root of Carcinogenesis
[https://www.frontiersin.org/articles/10.3389/fcell.2018.00090/ful](url)
